typings/index.d.ts

```diff
@@ -2825,12 +2825,15 @@ export function WebhookMixin<T>(Base?: Constructable<T>): Constructable<T & Webh
 export interface PartialWebhookFields {
   id: Snowflake;
   readonly url: string;
-  deleteMessage(message: MessageResolvable | APIMessage | '@original'): Promise<void>;
+  deleteMessage(message: MessageResolvable | APIMessage | '@original', threadId?: Snowflake): Promise<void>;
   editMessage(
     message: MessageResolvable | '@original',
     options: string | MessagePayload | WebhookEditMessageOptions,
   ): Promise<Message | APIMessage>;
-  fetchMessage(message: Snowflake | '@original', cache?: boolean): Promise<Message | APIMessage>;
+  fetchMessage(
+    message: Snowflake | '@original',
+    cacheOrOptions?: WebhookFetchMessageOptions | boolean,
+  ): Promise<Message | APIMessage>;
```

Comment:
If tslint lets you, splitting this overload out would be nice so that you can `@deprecated` the boolean overload

Reply:
I was only able to do this by disabling the `unified-signatures` rule.